generic/tests/netperf.py

```diff
@@ -120,19 +121,23 @@ def mtu_set(mtu):
             ssh_cmd(server_ctl, server_mtu_cmd % (ethname, mtu))
         elif params.get("os_type") == "windows":
             connection_id = utils_net.get_windows_nic_attribute(
-                            server_ctl, "macaddress", mac, "netconnectionid")
+                server_ctl, "macaddress", mac, "netconnectionid")
             ssh_cmd(server_ctl, server_mtu_cmd % (connection_id, mtu))
 
         error_context.context("Changing the MTU of client", logging.info)
-        ssh_cmd(client, client_mtu_cmd % (params.get("client_physical_nic"), mtu))
+        ssh_cmd(client, client_mtu_cmd
+                % (params.get("client_physical_nic"), mtu))
 
         netdst = params.get("netdst", "switch")
         host_bridges = utils_net.Bridge()
         br_in_use = host_bridges.list_br()
         if netdst in br_in_use:
             ifaces_in_use = host_bridges.list_iface()
             target_ifaces = list(ifaces_in_use + br_in_use)
-        error_context.context("Change all Bridge NICs MTU to %s" % mtu, logging.info)
+        if vm.virtnet[0].nettype == "macvtap":
+            target_ifaces.extend([params.get("netdst_nic1"), vm.get_ifname(0)])
```

Comment:
I guess the `params.get("netdst_nic1")` can be replaced with `vm.virtnet[0].netdst`.

Reply:
Updated and test passed and checked the result is as expected. 

23:19:21 DEBUG| Running 'ifconfig p5p2 mtu 9000'
23:19:21 DEBUG| Running 'ifconfig t0-2jMVXU mtu 9000'


JOB LOG    : /root/avocado/job-results/job-2018-03-26T23.17-e986af3/job.log
(1/2) smp_8.8192m.repeat1.repeat1.Host_RHEL.m7.u5.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.netperf.jumbo.host_guest: PASS (240.84 s)
 (2/2) smp_8.8192m.repeat1.repeat1.Host_RHEL.m7.u5.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.netperf.default.host_guest: PASS (289.83 s)